src/sass/components/_tour.scss

```diff
@@ -37,14 +37,73 @@
 
 .popover-wrapper {
   position: absolute;
-  top: -10px;
+  left: -($spacing-xx-small);
+  top: -($spacing-small);
+
+  .epic-progress & {
+    top: -($spacing-medium);
+  }
 
   .logo-popover & {
-    top: unset;
+    top: $spacing-small;
+    left: $spacing-large;
+  }
+
+  .metecho-breadcrumb & {
+    left: $spacing-large;
+    top: -($var-spacing-x-small);
+  }
+
+  .project-placeholder & {
+    top: $spacing-medium;
+  }
+
+  .project-list & {
+    left: $spacing-none;
+    top: $spacing-xxx-small;
   }
 
   .page-title & {
-    top: 25px;
+    left: $spacing-large;
+    top: $var-spacing-medium;
+  }
```

Comment:
@dvdherron we can try `.page-title > .popover-wrapper` to only target the direct children of the page-title container